### DISCUSSION TITLE
Blender on Android

### DISCUSSION THREAD

**Discussion by Kaimodo** [Yisus7u7/termux-desktop-lxqt#25] [Q&A] [answered]
I own a Shield K1. I was able to Install termux-desktop on it. But i didn't find a way to use QTerminal to Install Blender. I have to say i am used to the Darkside(Windows) normaly. I tryed apt but No Luck. I found synaptic pkg Manager, but No Luck there, too. Ist there any way to get it to Work? Thx in advance.

**Comment by JesusChapman** [comment, marked as answer]
> I own a Shield K1. I was able to Install termux-desktop on it. But i didn't find a way to use QTerminal to Install Blender. I have to say i am used to the Darkside(Windows) normaly. I tryed apt but No Luck. I found synaptic pkg Manager, but No Luck there, too. Ist there any way to get it to Work? Thx in advance.

Blender is not yet available for android.
